source/shared/smart-contracts/guides/invoke-instance.rst

```diff
@@ -0,0 +1,177 @@
+.. _invoke-instance:
+
+================================
+Invoke a smart contract instance
+================================
+
+This guide explains how to invoke a smart contract instance, which
+means triggering a receive function and viewing its return value. Invoking an
+instance is *not a transaction* and it *does not change the state of a contract*.
+Invoking can be useful to either view information about the instance or to test
+a receive method before running an update.
+The view functions are regular receive functions that return information about
+the contract, ideally with schemas on the return values.
+
+Preparation
+===========
+
+Make sure that you are :ref:`running a node<run-a-node>` using the latest :ref:`Concordium software<downloads>` and that you have a
+smart-contract instance on-chain to inspect.
+
+.. seealso::
+   For how to deploy a smart contract module see :ref:`deploy-module` and for
+   how to create an instance :ref:`initialize-contract`.
+
+
+Invocation
+==========
+
+To invoke an instance with address index ``0`` using the parameterless
+entrypoint ``my_receive``, run the following command:
+
+.. code-block:: console
+
+   $concordium-client contract invoke 0 --entrypoint my_receive
+
+If successful, the output should be similar to the following:
+
+.. code-block:: console
+
+   Successfully updated contract instance {"index":0,"subindex":0} using the entrypoint 'my_receive'.
+   Invocation resulted in success:
+   - Energy used: 595 NRG
+   - Return value:
+       <some-return-value>
+
+As you can see, the subindex defaults to ``0``.
+
+.. note::
+
+   By default, the invocation is sent by a fictional account with address ``00000000000000000000000000000000``.
```

Comment:
Shouldn't we use a base58 address here, maybe like "fictional account with address XXXX (the byte representation is [0u8; 32])?